oaknational/Oak-Web-Application
# Fix/lesq 627/landscape worksheets [LESQ-627]

## Description

Music year: 1958

- updates 2023 lesson overview MV to include worksheetIsLandscape field in line with 2020 api 

## Issue(s)

Fixes #
Worksheets not appearing in ladscape mode for migrated EYFS content

## How to test

1. Go to https://deploy-preview-2244--oak-web-application.netlify.thenational.academy
2. Click on any new lesson
3. The worksheet should display correctly

## Screenshots

How it used to look (delete if n/a):
<img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/cb1a26ce-484f-449f-9e4e-b15c46a973a4" width=500>


How it should now look:
<img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/2c65604e-b480-4132-a5d5-02585c4f0f1b" width=500 >
